calico/felix/config.py

```diff
@@ -248,6 +248,12 @@ def __init__(self, config_path):
                            "spoofing their source IP.  (For example, "
                            "unprivileged containers.)",
                            False, value_is_bool=True)
+        self.add_parameter("ClusterGUID", "Unique cluster identifier",
+                           "baddecaf")
+        self.add_parameter("UsageReportingEnabled",
+                           "If set to true, the cluster existence will"
```

Comment:
This would read better in active voice ('report X', vs 'X will be reported').  It should also have a little bit more explanation I think.  This (/ the configuration.rst copy of the same text) is the face of this change.

We should probably say a little more about what's reported, and why someone might want / want not to have it on.
- How about 'report anonymized cluster information to projectcalico.org, and receive deprecation / security warnings.'
- Privacy policy?  Talk to me.